moveit_core/robot_model/test/test.cpp

```diff
@@ -128,6 +128,50 @@ TEST(SiblingAssociateLinks, SimpleYRobot)
   }
 }
 
+TEST(FloatingJointTest, interpolation_test)
+{
+  // Create a simple floating joint model with some dummy parameters (these are not used by the test)
+  moveit::core::FloatingJointModel fjm("joint", 0, 0);
+
+  double jv1[7];
+  double jv2[7];
+  double intp[7];
+  random_numbers::RandomNumberGenerator rng;
+
+  for (size_t i = 0; i < 1000; i++)
```

Comment:
```suggestion
  for (size_t i = 0; i < 1000; ++i)
```

Pre-increment is slightly preferred for speed (although the compiler _probably_ takes care of it). Reference:  https://betterprogramming.pub/stop-using-i-in-your-loops-1f906520d548